src/pages/SignUp.jsx

```diff
@@ -0,0 +1,74 @@
+import { Link } from "react-router-dom";
+import authbg from "../assets/auth_bg.png";
+
+const SignUp = () => {
+     return (
+          <>
+               <div className="h-screen flex flex-row">
+                    <div className="h-full w-full md:w-1/2 flex justify-center items-center">
+                         <div className="h-fit w-full md:w-[450px] flex flex-col justify-center gap-5 md:gap-10">
+                              <h1 className="font-normal text-md md:text-lg text-white font-Bayon text-center">Welcome to <span className="text-primary-default">Will Be There</span></h1>
+                              <form noValidate className="flex flex-col gap-5 md:gap-10">
+                                   <div className="px-5 flex flex-col gap-5 md:gap-10">
+                                        {/* <div className="flex flex-col gap-10"> */}
+                                        <div className="flex flex-col font-Bayon">
+                                             <h2 className="text-base md:text-md text-start uppercase text-primary-default">sign up</h2>
+                                             <p className="text-slate uppercase font-Bayon font-normal text-sm md:text-base">have an account? <Link to="/login" className="text-primary-default">login</Link></p>
+                                        </div>
+                                        {/* inputs */}
+                                        <div className="w-full flex flex-col font-Bayon gap-5 md:gap-[37px]">
+                                             <input type="text" name="first_name" required autoComplete="none" placeholder="enter user name"
+                                                  className="w-full flex text-primary-default text-sm md:text-base  uppercase rounded-[10px] border border-white outline-none  placeholder:text- p-[10px] placeholder:text-primary-default"
+                                                  onInvalid={(e) => e.target.setCustomValidity("Please input first name here",)}
+                                                  onInput={(e) => e.target.setCustomValidity("")} />
+                                             <input type="email" name="email_address" autoComplete="none" placeholder="enter email"
+                                                  className="w-full flex text-primary-default text-sm md:text-base  uppercase rounded-[10px] border border-white outline-none  placeholder:text- p-[10px] placeholder:text-primary-default"
+                                                  required />
+                                             {/* </div> */}
+                                             <div className="flex flex-col gap-3 md:gap-5">
+                                                  <input type="password" name="password" placeholder="enter at lease 6 symbols"
```

Comment:
Fix the typo "least" not "lease". Also, create a another input field for password confirmation

Reply:
fix

Reply:
fixed.